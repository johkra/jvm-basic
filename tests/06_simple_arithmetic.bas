PRINT 13+22
PRINT -13+22
PRINT 22-13
PRINT 13*22
PRINT 39/13
PRINT 12^4
