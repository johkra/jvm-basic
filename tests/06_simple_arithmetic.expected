35
9
9
286
3
20736
